doc/classes/ConvexPolygonShape.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ConvexPolygonShape" inherits="Shape" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Convex Polygon Shape.
+		Convex Polygon Shape for 3D physics.
```

Comment:
The class name is capitalized, but the words wouldn't be in a description:

    Convex polygon shape for 3D physics.